backend/src/main/java/com/bakdata/conquery/io/storage/xodus/stores/XodusStore.java

```diff
@@ -1,23 +1,25 @@
 package com.bakdata.conquery.io.storage.xodus.stores;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Spliterator;
+import java.util.Spliterators;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
-import java.util.function.BiConsumer;
 import java.util.function.Consumer;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
 
 import com.google.common.primitives.Ints;
 import jetbrains.exodus.ByteIterable;
 import jetbrains.exodus.env.Cursor;
 import jetbrains.exodus.env.Environment;
 import jetbrains.exodus.env.Store;
 import jetbrains.exodus.env.StoreConfig;
+import jetbrains.exodus.env.Transaction;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
-public class XodusStore {
+public class XodusStore implements com.bakdata.conquery.io.storage.Store<ByteIterable, ByteIterable> {
```

Comment:
Das implementieren des stores scheint hier nichts zu machen, weil du die meisten methoden einfach leer lässt?

Reply:
So konnte ich ihn einfach in einen CachedStore wappen